### PR TITLE
Updated Readme file and added pull request template(#2q9nxqt)

### PR DESCRIPTION
**Pull request by azkyakhan**

### CODE REVIEW

File: .github/workflows/firebase-hosting-merge.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# This file was auto-generated by the Firebase CLI
```
Comment (adityasharma7) [inline .github/workflows/firebase-hosting-merge.yml:1]: This is not needed

File: .github/workflows/pull-request.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install Dependencies
+      run: npm install
+    - name: Generate .env file
+      run: cp .env.example .env
+    - name: Build
+      run: npm run build
```
Comment (adityasharma7) [inline .github/workflows/pull-request.yml:19]: Add `buildScript` along with build

**Review by adityasharma7 — CHANGES_REQUESTED** [review]

File: .github/workflows/firebase-hosting-merge.yml (hunk 1 above)
Comment (azkyakhan) [reply .github/workflows/firebase-hosting-merge.yml]: Removed this line sir

File: .github/workflows/pull-request.yml (hunk 2 above)
Comment (azkyakhan) [reply .github/workflows/pull-request.yml]: I have added `buildScript` along with build

**Review by adityasharma7 — APPROVED** [review]

**Pull request merged**